test/integration/migration.spec.js

```diff
@@ -171,19 +172,60 @@ describe('the migration', function () {
     ]);
   }));
 
+  it('changes field IDs', co(function * () {
+    yield migrator(idChange);
+
+    const resultAfterModify = yield request({
+      method: 'GET',
+      url: '/content_types/dog',
+      headers: {
+        'X-Contentful-Beta-Dev-Spaces': 1
+      }
+    });
+
+    expect(resultAfterModify.fields).to.eql([
+      {
+        id: 'goodboys',
+        name: 'ID switching is fun!',
+        type: 'Number',
+        required: false,
+        disabled: false,
+        localized: false,
+        omitted: false,
+        validations: []
+      }
+    ]);
+
+    let thisShouldNotExist;
+
+    try {
+      thisShouldNotExist = yield request({
+        method: 'GET',
+        url: '/content_types/aDifferentId',
```

Comment:
You cannot rename a Content Type, so this can be dropped.

Reply:
Um yes, I meant to get the old field here, not a content type. Removing it.